### PR TITLE
removing backup_storage_hook from the doc

### PR DESCRIPTION
**Pull request by rsajwani**
Signed-off-by: Rameez Sajwani <rameezwazirali@hotmail.com>

Fixes Issue https://github.com/vitessio/website/issues/1237

removing deprecated flags from backup.

### CODE REVIEW

**Review by maxenglander — APPROVED** [review]

File: content/en/docs/16.0/reference/programs/vtbackup.md [hunk 1]
```diff
@@ -22,7 +22,7 @@ Whatever system launches `vtbackup` is responsible for the following:
 
 ## Example Usage
 
-The following example launches the `vtbackup`.
+Consider you have running Vitess cluster, the following command will create backup using `vtbackup` for keyspace `commerce` having shard `0`
```
Comment (deepthi) [inline content/en/docs/16.0/reference/programs/vtbackup.md]:
```suggestion
On a running Vitess cluster, the following command will create a backup using `vtbackup` for keyspace `commerce` and shard `0`.
```

Comment (deepthi) [reply content/en/docs/16.0/reference/programs/vtbackup.md]: The previous version didn't sound right when I read it out.

File: content/en/docs/16.0/user-guides/operating-vitess/backup-and-restore/overview.md [hunk 2]
```diff
@@ -76,28 +76,21 @@ All three programs can be made aware of Backup Engine and Backup Storage using t
...
         By default it is set to true. Use
         <code>--backup_storage_compress=false</code> to disable.</br>
-        This is meant to be used with a <code>--backup_storage_hook</code>
-        hook that already compresses the data, to avoid compressing the data
-        twice.
+      </td>
+    </tr>
+    <tr>
+      <td><code>backup_storage_block_size</code></td>
+      <td>If <code>--backup_storage_compress</code> is true, <code>backup_storage_block_size</code> sets the byte size for each block while compressing (default is 250000).
```
Comment (deepthi) [inline content/en/docs/16.0/user-guides/operating-vitess/backup-and-restore/overview.md]:
```suggestion
      <td>If <code>--backup_storage_compress</code> is true, <code>backup_storage_block_size</code> sets the block size in bytes to use while compressing (default is 250000).
```

File: content/en/docs/16.0/user-guides/operating-vitess/backup-and-restore/overview.md [hunk 3]
```diff
@@ -76,28 +76,21 @@ All three programs can be made aware of Backup Engine and Backup Storage using t
...
+      </td>
+    </tr>
+    <tr>
+      <td><code>backup_storage_number_blocks</code></td>
+      <td>If <code>--backup_storage_compress</code> is true, <code>backup_storage_number_blocks</code> sets the number of blocks that can be processed, at once, before the writer blocks, during compression. It should be equal to the number of CPUs available for compression. (default 2)
```
Comment (deepthi) [inline content/en/docs/16.0/user-guides/operating-vitess/backup-and-restore/overview.md]: at once or in parallel? Not sure I understand what this flag is doing from the description.

**Review by deepthi — COMMENTED** [review]
@rsajwani can you please address the feedback before you merge this?

File: content/en/docs/16.0/user-guides/operating-vitess/backup-and-restore/overview.md (hunk 3 above)
Comment (rsajwani) [reply content/en/docs/16.0/user-guides/operating-vitess/backup-and-restore/overview.md]: I copied the text from help. Verified the behavior , yes it should be 'parallel.. I will fix the help output text as well..

**Pull request merged**